src/mainwindow.cpp

```diff
@@ -1,12 +1,25 @@
 #include <QWebFrame>
+#include <QStandardPaths>
 
 #include "mainwindow.h"
 #include "view.h"
 
 
 MainWindow::MainWindow() : QMainWindow()
 {
+    QWebSettings::globalSettings()->setAttribute(QWebSettings::LocalStorageEnabled, true);
+
+    manager = new QNetworkAccessManager(this);
+    diskCache = new QNetworkDiskCache(this);
```

Comment:
there is no point in making those variables a class member variables. They are not used anywhere and you can always get them back. Just declare them in the same function